pom.xml

```diff
@@ -141,7 +141,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.18</version>
+            <!-- Temp SNAPSHOT from https://github.com/jenkinsci/workflow-job-plugin/pull/95 -->
+            <version>2.20-20180419.201134-1</version>
```

Comment:
But it will also work without this update, right?

Reply:
No, `getOrNull` is semi-hosed without this update.

Reply:
IIUC with the released `workflow-job` the **Replay** link would be unconditionally available on a build whose execution was not yet loaded, and then when you click on it, you either get the replay form, or you get a rebuild form if you for example lacked `RUN_SCRIPTS`. Right? Or was there an NPE interfering?

Reply:
Unconditionally available but then you'd never actually be able to do the replay because you'd always have null, so... oops?